app/workers/notify_vacancy_on_slack_worker.rb

```diff
@@ -0,0 +1,16 @@
+require "slack_notifier"
+
+class NotifyVacancyOnSlackWorker
+  include Sidekiq::Worker
+  include Rails.application.routes.url_helpers
+
+  def perform(vacancy_id)
+    vacancy = Vacancy.find(vacancy_id)
+    message = %Q(
```

Comment:
Like we discussed, the `SlackNotifier` class should not be coupled with the Vacancy. Maybe creating a class to generate the message. What do u think @lastk ?